kernel/arch/dreamcast/sound/snd_stream.c

```diff
@@ -658,12 +616,16 @@ void snd_stream_stop(snd_stream_hnd_t hnd) {
         cmd->cmd_id = streams[hnd].ch[1];
         snd_sh4_to_aica(tmp, cmd->size);
     }
+
+    if(streams[hnd].channels == 2) {
+        snd_sh4_to_aica_start();
```

Comment:
Can this be added to the if statement above on line 614?

Reply:
lol, of course, fixed!